ui/raidboss/popup-text.ts

```diff
@@ -718,7 +718,7 @@ export class PopupText {
           // time later.  This will clobber each time we
           // load this, but that's ok.
           trigger.filename = setFilename;
-          const id = trigger.id;
+          const id = trigger.id ?? trigger.toString();
```

Comment:
here `id` is optional, and it's option in user trigger, I don't know how to handle there...

Reply:
And I found it's a much more complicated type issue here https://github.com/quisquous/cactbot/issues/5039

fixing the `LooseTriggerSet` cause many lint error.